public_html/wp-content/mu-plugins/disable-modules.php

```diff
@@ -0,0 +1,21 @@
+<?php
```

Comment:
I think this would be more consistent in `mu-plugins/jetpack-tweaks/miscellaneous.php` or `jetpack-tweaks/new-sites.php`. wordcamp.org tends to organize mu-plugins into categories, rather than make lots of small ones like w.org. That feels less cluttered IMO.

`new-sites.php` isn't an exact fit, but we already have a `jetpack_get_default_modules` callback there, and having these two callbacks side by side would make it more obvious that both exist.

https://github.com/WordPress/wordcamp.org/blob/f8d1ba751855f389984077af1f10a464af91d916/public_html/wp-content/mu-plugins/jetpack-tweaks/new-sites.php#L13-L21

Alternatively, moving it inside the `jetpack-tweaks/` folder would help keep it organized. Not a blocker, though.

Reply:
I had forgotten about the `jetpack-tweaks/` folder.  That location seems ideal.

Reply:
Ah, looks like it was already moved in fa2f6e4e73af8dc0bbed0831b7a2f6f3bd90b250 👍🏻